ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
```

Comment:
i'm fine with a high-level, one-to-two sentence for 0.2.0. personally i think the milestone tagging stuff is a bit too much process and not that important, but you can